crates/bevy_core_pipeline/src/tonemapping/mod.rs

```diff
@@ -54,9 +79,31 @@ pub struct TonemappingPipeline {
     texture_bind_group: BindGroupLayout,
 }
 
+#[derive(Default, Debug, Copy, Clone, PartialEq, Eq, Hash, Reflect, FromReflect)]
+#[reflect(FromReflect)]
+pub enum TonemappingMethod {
+    None,
+    /// Suffers from lots hue shifting, brights don't desaturate naturally.
+    Reinhard,
+    /// Old bevy default. Suffers from hue shifting, brights don't desaturate much at all.
+    ReinhardLuminance,
+    /// Bad
+    Aces,
```

Comment:
Are there different kinds of ACES? I've heard of filmic, and fitted. The Uncharted version too.

Reply:
It seems this is a 'fitted' version from Godot. I feel like we should maybe rename this to `AcesFitted`?